addons/stock/static/src/stock_forecasted/stock_forecasted.js

```diff
@@ -0,0 +1,124 @@
+/** @odoo-module **/
+
+import { useService } from "@web/core/utils/hooks";
+import { registry } from "@web/core/registry";
+import { View } from "@web/views/view";
+import { ControlPanel } from "@web/search/control_panel/control_panel";
+
+import { ForecastedButtons } from "./forecasted_buttons";
+import { ForecastedDetails } from "./forecasted_details";
+import { ForecastedHeader } from "./forecasted_header";
+import { ForecastedWarehouseFilter } from "./forecasted_warehouse_filter";
+
+const { Component, onWillStart, useState, useSubEnv } = owl;
+
+class StockForecasted extends Component {
+    setup() {
+        useSubEnv({
+            ...this.env,
+            //ControlPanel trick : Allow the use of ControlPanel's bottom-right while disabling search to avoid errors
+            searchModel: {
+                searchMenuTypes: [],
+            },
+            config: {
+                ...this.env.config,
+                viewSwitcherEntries: [],
+            }
+        });
+
+        this.orm = useService("orm");
+        this.action = useService("action");
+
+        this.context = useState(this.props.action.context);
+        this.productId = this.context.active_id;
+        this.resModel = this.context.active_model;
+        const isTemplate = this.resModel === "product.template";
+        this.reportModelName = `stock.forecasted_product_${isTemplate ? "template" : "product"}`;
+        this.title = this.props.action.name;
+
+        onWillStart(this._getReportValues);
+    }
+
+    async _getReportValues() {
+        this.resModel = this.context.active_model || (this.context.params && this.context.params.active_model);
+        if (!this.resModel && this.props.action.res_model) {
+            const actionModel = await this.orm.read('ir.model', [Number(this.props.action.res_model)], ['model']);
+            if (actionModel.length && actionModel[0].model) {
+                this.resModel = actionModel[0].model
+            }
+        } else if (this.props.action._originalAction) {
+            const originalContextAction = JSON.parse(this.props.action._originalAction).context;
+            if (originalContextAction) {
+                this.resModel = originalContextAction.active_model
+            }
+        }
+        const isTemplate = !this.resModel || this.resModel === 'product.template';
+        this.reportModelName = `report.stock.report_product_${isTemplate ? 'template' : 'product'}_replenishment`;
```

Comment:
Causes a traceback because `report.stock.report_product_template` (or `report.stock.report_product_product`) doesn't exist anymore. I guess the `reportModelName` like defined in the `setup` is enough and this part can be removed